projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts

```diff
@@ -24,7 +24,7 @@ import { OFormControl } from '../o-form-control.class';
 import {
   DEFAULT_INPUTS_O_FORM_SERVICE_COMPONENT,
   DEFAULT_OUTPUTS_O_FORM_SERVICE_COMPONENT,
-  OFormServiceComponent
+  OFormServiceComponent,
```

Comment:
can this trailing comma cause tslint problems?

Reply:
I dont think so, but I had the typescript hero plugin misconfigured and it was adding this comma.